ocaml/xapi/valid_ref_list.ml

```diff
@@ -17,4 +17,3 @@ let map f = Stdext.Listext.List.filter_map (default_on_missing_ref (fun x -> Som
 let flat_map f l = List.map (default_on_missing_ref f []) l |> List.flatten
 
 let filter_map f l = map f l |> List.filter ((<>) None) |> List.map Xapi_stdext_monadic.Opt.unbox
-
```

Comment:
I think this somehow happened while rebasing? It's unrelated so I can remove it, though I'm not sure why we have a blank line there at all.

Reply:
Yes, this happened when we resolved the merge conflicts. If you'd like to, you can undo this by adding back the blank line and then doing a `fixup` in [git interactive rebase](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History).

Sometimes people consider a blank line at the end of file useful, because then we can concatenate them with `cat`, and the last line of a file won't be merged into the first line of the next file.